src/MensaWidget/MWFrame.class.st

```diff
@@ -0,0 +1,54 @@
+"
+A MWFrame is the base block for the useer interface of the widget.
+
+Instance Variables
+
+"
+Class {
+	#name : #MWFrame,
+	#superclass : #PluggableSystemWindow,
+	#category : #MensaWidget,
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 19:43'
+}
+
+{
+	#category : #'as yet unclassified',
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 17:09'
+}
+MWFrame >> addText [
+
+	| textMorph |
+	
+	textMorph := TextMorph new.
+		textMorph contents: 'Are you a Munch? ;)'.
+		textMorph position:  135@100.
+		
```

Comment:
Do we want to agree on doing breaklines before self?

Reply:
no i mistakenly did

Reply:
done